lesson05/banking-application/bank.js

```diff
@@ -0,0 +1,78 @@
+let balance = 1000;
+
+function start() {
+    let keepGoing = true;
+    while (keepGoing) {
+        keepGoing = askUserInput();
+    }
+}
+
+function askUserInput() {
+   const response = prompt('Enter Q to quit. Enter W to withdraw. Enter D to deposit. Enter B to view balance.');
+   switch(response) {
+       case "Q": case "q":
```

Comment:
Instead of listing both cases, the best way of ensuring both options are covered would be converting your user's response to uppercase or lowercase `switch(response.toUpperCase()`